Command/ClearChunkCommand.php

```diff
@@ -2,14 +2,24 @@
 
 namespace Oneup\UploaderBundle\Command;
 
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Oneup\UploaderBundle\Uploader\Chunk\ChunkManager;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class ClearChunkCommand extends ContainerAwareCommand
+class ClearChunkCommand extends Command
 {
     protected static $defaultName = 'oneup:uploader:clear-chunks'; // Make command lazy load
 
+    /** @var ChunkManager */
+    protected $manager;
+
+    public function __construct(ChunkManager $manager, ?string $name = null)
```

Comment:
New argument was introduced. It should be passed to command in `Resources/config/uploader.xml`.